### PR TITLE
ExpirableSet added and updated .proto to 1.4.0

### PR DESCRIPTION
**Pull request by raducrisan1**

### CODE REVIEW

File: ImmuDB4Net.Tests/BasicClientIntegrationTests.cs [hunk 1]
```diff
@@ -152,6 +152,27 @@ public async Task TestSimpleVerifiedSetAndGet()
...
+    
+    [TestMethod("execute expirableset and verifiedget")]
+    public async Task TestExpirableSet()
+    {
+        await client!.Open("immudb", "immudb", "defaultdb");
+
+        byte[] v0 = new byte[] { 0, 1, 2, 3 };
+        byte[] v1 = new byte[] { 3, 2, 1, 0 };
+
+        TxHeader hdr0 = await client.ExpirableSet("k0", v0, DateTime.Now.AddDays(1));
```
Comment (byo) [inline ImmuDB4Net.Tests/BasicClientIntegrationTests.cs:168]: Can we also add a test when the key expires? like:

```go
TxHeader hdr0 = await client.ExpirableSet("k0", v0, DateTime.Now.AddDays(-1));
```

Reading such entry should end up with `key not found` error.

File: ImmuDB4Net/proto/schema.proto [hunk 2]
```diff
@@ -30,82 +30,131 @@ message Key {
 }
 
 message Permission {
+  // Database name
```
Comment (byo) [inline ImmuDB4Net/proto/schema.proto:33]: Something's weird with the indentation here - is it also in the original .proto file ?

**Review by byo — APPROVED** [review]
LGTM with some minor comments

File: ImmuDB4Net.Tests/BasicClientIntegrationTests.cs (hunk 1 above)
Comment (raducrisan1) [reply ImmuDB4Net.Tests/BasicClientIntegrationTests.cs:168]: yes, done.

File: ImmuDB4Net/proto/schema.proto (hunk 2 above)
Comment (raducrisan1) [reply ImmuDB4Net/proto/schema.proto]: done

**Pull request merged**